common/src/main/java/io/netty/util/NetUtil.java

```diff
@@ -134,7 +134,8 @@
         // Create IPv4 loopback address.
         Inet4Address localhost4 = null;
         try {
-            localhost4 = (Inet4Address) InetAddress.getByAddress(LOCALHOST4_BYTES);
+//          localhost4 = (Inet4Address) InetAddress.getByAddress("localhost", LOCALHOST4_BYTES);
```

Comment:
sorry, i commented out the fix to copy the error message from the unit test for the issue description and committed that instead of the fix. i have overwritten the correct commit now